### ISSUE TITLE
Bugs

### ISSUE THREAD

**Issue by marciesdr** [eFiction/v5_1-vaporware#1]
1) During install if you don't install the contest module you will get an error in the backend when installation is complete. The Archive area will be absent and won't display the categories within it.  Not sure if this has to do with the installer or vaporware. But most likely the installer since I didn't install this. So must add option to somehow install it or give feedback on how to implement it back in. Hence this bug:

Undefined index: contests

[fiction/app/model/admincp.php(247) : eval()'d code:1] Base->{closure}()
[fiction/app/model/admincp.php:247] eval()
[fiction/app/controller/admincp.php:27] Model\AdminCP->showMenu()
[fiction/app/controller/admincp.php:22] Controller\AdminCP->showMenu()
[fiction/index.php:117] Base->run()

*** This one is resolved I just added the two tables: Contest and Contest_Relations  and input the setting into the config via the optional modules table. {"recommendations":1,"contests":1,"shoutbox":1,"poll":1,"tracker":1}**

2) Can't post news can add headline but can't post the actual news and even though the date is right when you post it it shows wrong on the frontend. Hence this error:

Call to a member function format() on boolean [/home/a123456/public_html/site/fiction/app/model/admincp.php:1086]

[fiction/app/controller/admincp_home.php:246] Model\AdminCP->saveNews()
[fiction/app/controller/admincp_home.php:23] Controller\AdminCP_Home->news()
[fiction/index.php:117] Base->run()

3) Can post stories fine but no updates show not sure why maybe cause i just imported it but word count works then it just lists 1.

4) The categories widget shows but when click no actual stories are produced.

5) User control panel can't add story chapter. does this mean I need to add a table cause in the admin panel I can add a new chapter. So must be error in the coding of the user control panel.

PDOStatement: Table 'asd123456_fic.new5_users' doesn't exist

[fiction/app/model/base.php:19] DB\SQL->exec()
[fiction/app/model/base.php:307] Model\Base->exec()
[fiction/app/model/usercp.php:533] Model\Base->storyChapterAdd()
[fiction/app/controller/usercp.php:221] Model\UserCP->authorStoryChapterAdd()
[fiction/app/controller/usercp.php:103] Controller\UserCP->authorStoryEdit()
[fiction/app/controller/usercp.php:34] Controller\UserCP->author()
[fiction/index.php:117] Base->run()

**Comment by campZero** [comment]
Four bugs were fixed or could not be reproduced (news issue):

1) Fixed, would always evaluate a value even if not present, now added a check

2) I could post a news entry including headline and text when testing. There was an issue with the date/time picker, I updated the java and made the sricpt remove multiple whitespaces leading to the user not being able to change the date.

5) Hard coded table prefix, the most stupid of all mistakes  :omg:  The userCP version of story edit is much more recent than the adminCP counterpart, so there's gonna be a major transfer to do (see 6)

6) Was an issue with stories that do not yet have a chapter, hencer no chapter list. Most my testing so far happend on existing data so that didn't happen. Now fixed. Parts of the codes and logic must be updated with the userCP counterpart.
